src/main/mulmo/handler_contents.ts

```
@@ -95,3 +100,36 @@
+};
+
+export const mulmoRefecenceImagesFiles = async (projectId: string) => {
+  const context = await getContext(projectId);
+  const images = context.presentationStyle.imageParams?.images;
+  if (!images) {
+    return {};
+  }
+  const imageRefs: Record<string, string> = {};
+  Object.keys(images)
+    .sort()
+    .map((key, index) => {
```

ESLint

[error] 'index' is defined but never used. Allowed unused args must match /^__/u.

(@typescript-eslint/no-unused-vars)